### PR TITLE
Activate security products

### PR DESCRIPTION
**Pull request by robertpi**

### CODE REVIEW

File: test.sh
Comment (bkabrda) [inline test.sh:1]: Was this file added by mistake? I don't see it being used anywhere.

File: install_script.sh.template [hunk 1]
```diff
@@ -1419,6 +1436,8 @@ elif [ ! "$no_agent" ]; then
   manage_security_and_system_probe_config "$sudo_cmd" "$security_agent_config_file" "$system_probe_config_file" "$DD_RUNTIME_SECURITY_CONFIG_ENABLED" "$DD_COMPLIANCE_CONFIG_ENABLED"
 fi
 
+manage_client_libraries_security_config "$sudo_cmd" "$environment_file" "$DD_APPSEC_ENABLED" "$DD_IAST_ENABLED" "$DD_APPSEC_SCA_ENABLED"
```
Comment (bkabrda) [inline install_script.sh.template:1439]: Nit: two variables names start with `DD_APPSEC_` and one only with `DD_`. Is this intentional? Would it make more sense to make them all start with `DD_APPSEC_`?

File: test.sh
Comment (robertpi) [reply test.sh:1]: Gosh, that's a silly mistake. Remove it in bb31bb0

File: install_script.sh.template (hunk 1 above)
Comment (robertpi) [reply install_script.sh.template:1439]: While I agree with your comments, these settings have be implemented and document in the client libraries for quite sometime, so are not possible to change without requesting all customers using them to migrate.

File: unit_tests/test_install_script.sh [hunk 2]
```diff
@@ -249,5 +250,27 @@ testFullConfig(){
   assertEquals 1 $?
 }
 
+### Manage client libraries security config
+testManageClientLibrariesSecurityConfig() {
+  rm $environment_file
```
Comment (spencergilbert) [inline unit_tests/test_install_script.sh:255]: [Small note that this sort of errors, or at least just emits an unneeded log](https://gitlab.ddbuild.io/DataDog/agent-linux-install-script/-/jobs/460233248#L512) - don't expect it to be a blocker/problem, just unexpected for me to see it when I was reviewing the test logs.

**Review by spencergilbert — APPROVED** [review]

**Pull request merged**